include/Network/HTTP/HTTPException.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file include/Network/HTTP/HTTPException.php
+ *
+ * @brief Throwable exceptions to return HTTP status code
+ *
+ * This list of Exception has be extracted from
+ * http://racksburg.com/choosing-an-http-status-code/
+ */
+
+namespace Friendica\Network\HTTP;
+
+/**
+ * @brief Base class for HTTP response codes
+ */
+class HTTPException extends \Exception 
+{
+
+	var $httpcode = 200;
+	var $httpdesc = "";
+
+	/**
+	 * @brief Create a new HTTPException
+	 *
+	 * @param string $message Error message
+	 * @param int $code Error code
+	 * @param Exception $previous Previous exception
+	 */
+	public function __construct($message="", $code = 0, Exception $previous = null) {
+		if ($this->httpdesc=="") {
+			$this->httpdesc = preg_replace("|([a-z])([A-Z])|",'$1 $2',
+								str_replace("Exception","",
```

Comment:
This indentation doesn't look right with a different tab size than you use. Please only use a single indentation for functions parameters and add a space after each comma.